package.json

```diff
@@ -23,14 +25,26 @@
     "isomorphic-fetch": "^2.2.1",
     "joi-browser": "^10.0.6",
     "node-fetch": "^1.6.3",
-    "uuid": "^3.0.0"
+    "uuid": "^3.0.1"
   },
   "devDependencies": {
+    "babel": "^6.5.2",
+    "babel-core": "^6.22.1",
+    "babel-loader": "^6.2.10",
+    "babel-polyfill": "^6.22.0",
+    "babel-preset-es2015": "^6.22.0",
     "chai": "^3.5.0",
     "chai-as-promised": "^6.0.0",
+    "coveralls": "^2.11.16",
     "eslint": "^3.12.2",
     "fetch-mock": "^5.5.0",
+    "istanbul": "^0.4.5",
+    "karma": "^1.4.1",
+    "karma-firefox-launcher": "^1.0.0",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-reporter": "^2.2.2",
+    "karma-webpack": "^2.0.2",
     "mocha": "^3.1.2",
-    "npm-run-all": "^3.1.1"
+    "webpack": "^1.14.0"
```

Comment:
Yea, the latest version wasn't playing nicely with babel. I tried some other version that played nicely with babel but not karma O_o.... this version seemed to work with both...